File: app/src/main/java/com/jzoft/ygohelper/biz/ProxyCardLocator.java
package com.jzoft.ygohelper.biz;

import com.jzoft.ygohelper.utils.HttpCaller;

/**
 * Created by jjimenez on 11/10/16.
 */
public interface ProxyCardLocator {
    ProxyCard locate(String location) throws HttpCaller.NotFound;

    boolean hasToLocate(String location);

}

File: app/src/main/java/com/jzoft/ygohelper/biz/ProxyCardLocator.kt
package com.jzoft.ygohelper.biz

import com.jzoft.ygohelper.utils.Caller
import rx.Observable


/**
 * Created by jjimenez on 11/10/16.
 */
interface ProxyCardLocator {

    fun locate(location: String): Observable<ProxyCard>

    fun hasToLocate(location: String): Boolean

}
